src/js/components/Card/Card.js

```diff
@@ -0,0 +1,53 @@
+import React, { forwardRef, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Box } from '../Box';
+import { Header } from '../Header';
+import { Footer } from '../Footer';
+
+export const Card = forwardRef(({ children, header, footer, ...rest }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  return (
+    <Box align="start" justify="between" {...theme.card} {...rest}>
+      {header && (
+        <Header
```

Comment:
Hmm... something feels a bit odd here. What if the caller wants to change the `justify` on the Header? And, passing multiple contents via something like `<Card header={[<Avatar />, <Button />]} />` seems heavy

I think we are wanting Card to have specific default Header and Footer styles but allow them to be customized where needed. 

What about treating Card more like Table?:
```
<Card>
  <CardHeader />
  <CardBody />
  <CardFooter />
 </Card>
```
Where CardBody is only needed if you have CardHeader or CardFooter.
Just thinking.